docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,81 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+If you have [configured the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
```

Comment:
```suggestion
After [configuring the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
```
@npentrel might be better

Reply:
Ok I'm still confused about the ordering of decisions a reader has to make. I think this sentence should merge with the next paragraph to explain why you'd choose one or the other